### PR TITLE
ADR: Progression and unlocking system

### PR DESCRIPTION
**Pull request by delvh**

### CODE REVIEW

**Comment by delvh** [comment]
I dug out this branch which surprisingly never got a PR. 

@WuestFlorian, @cdingm one of you two is responsible for updating the branch with the results from the survey. Your choice, who does it.

**Review by Gr33ndev — APPROVED** [review]

**Comment by Gr33ndev** [comment]
please merge manually 2022-08-27_19h54_40

File: adr/2022-08-17-progression-system.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+3. all minigames in the current area need to be played and completed (`highscore >= x`)
+
+## Pro Chosen Solution
+
+- User knows exactly what to do
+- All study content has to be understood to a certain degree
+
+## Contra Chosen Solution
+
+- Knowledge doesn't really have a purpose now
```
Comment (delvh) [inline adr/2022-08-17-progression-system.md:25]:
```suggestion
- Knowledge doesn't really have a purpose now
- User might get stuck and need help with a minigame to progress
```

File: adr/2022-08-17-progression-system.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+6. all minigames in the current area need to be played and completed (`highscore >= x`) and average highscore of all minigame in all areas needs to be greater than `y`
+
+## Chosen Solution
+
+3. all minigames in the current area need to be played and completed (`highscore >= x`)
+
+## Pro Chosen Solution
+
+- User knows exactly what to do
+- All study content has to be understood to a certain degree
```
Comment (delvh) [inline adr/2022-08-17-progression-system.md:21]:
```suggestion
- All study content has to be understood to a certain degree
- No XP (knowledge) farming possible/ necessary
```

**Pull request closed without merging**